docs/learn/curvature-drive.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+title: "Curvature Drive"
+parent: "Learning"
+permalink: /docs/learn/curv
+authors: ['hyperliskdev']
+---
+
+# Curvature Drive
+
+Curvature drive is a method inside of the DifferentialDrive class in WPILib 
+
+## What does Curvature Drive do?
+
+Curvature drive is a type of drive just like Arcade or Tank drive. Curvature drive is different in a few key ways;
+
+- isQuickTurn method is used to quickturn on the stop, as it is default off.
+- Turning speed withput isQuickTurn == true, is based on how fast the robot is moving.
+- The robot can be driven with either Trigger Control or Joystick Control
+- And finally, the robot now drives like a car.
+
+## How to make Curvature drive
+
+```java
+
+# Inside of my drivetrain class.
+public void curvDrive(double speed, double rotation, boolean isQuickTurn) {
+    isMoving = (speed != 0.0);
+    isTurning = (rotation != 0.0);
+
+    m_DifferentialDrive.curvatureDrive(speed, rotation, isQuickTurn);
+    
+    
+  }
+```
+
+Is moving isnt required but could be helpful in auto or for something else ive never heard of.
+
+Inside the drive controller class:
+
+``` java
+// Called repeatedly when this Command is scheduled to run
+  @Override
+  protected void execute() {
+
+
+    // Main vars for driving.
+    speed = Robot.m_oi.getTriggerThrottle();
+    rotation = Robot.m_oi.getRotation();
+    isQuickTurn = Robot.m_oi.getQuickTurn();
+
+    // Deadbanding stuff
+    speed = m_speedDeadband.feed(speed);
+    rotation = m_rotationDeadband.feed(rotation);
```

Comment:
These have nothing to do with drive.. Did you just copy-paste our codebase into a MD file?